frontend/fingertips-frontend/components/forms/SearchForm/index.tsx

```diff
@@ -10,34 +17,57 @@ const StyledInputField = styled(InputField)(
   spacing.withWhiteSpace({ marginBottom: 6 })
 );
 
-export const SearchForm = ({ indicator }: { indicator: string }) => {
-  const initialState: SearchFormState = {
-    indicator: '',
-    message: null,
-    errors: {},
-  };
-  const [_state, formAction] = useActionState(searchIndicator, initialState);
+export const SearchForm = ({
+  searchFormState,
+}: {
+  searchFormState: SearchFormState;
+}) => {
+  const [state, formAction] = useActionState(searchIndicator, searchFormState);
 
   return (
-    <>
-      <H1>Search</H1>
+    <main>
```

Comment:
I don't think we need this to be `main` - The container component already does this. I think this was added at some point in some other PR so it could have been complaining about needing this before.

Reply:
Yes, think this went in from another PR. I notice the adding of landmarks however has added one (footer) to a component that already contains one (the govuk footer), should I remove it?

Reply:
ah yes try removing it. The accessibility test were complaining about the footer landmark being missing but you are right it already does contain a `footer` landmark as part of the govuk-footer